cedar-lean/Cedar/Spec/Ext/Decimal.lean

```diff
@@ -58,6 +58,19 @@ def parse (str : String) : Option Decimal :=
     else .none
   | _ => .none
 
+def unParse (d : Decimal) : String :=
```

Comment:
Why `unParse` and not `toString` or similar?

Reply:
No reason lol

Reply:
I'll do the ToString instance as a followup

Reply:
#286